defaults/main.yml

```diff
@@ -136,7 +136,7 @@ rke2_tarball_images_path: "{{ rke2_data_path }}/agent/images"
 rke2_architecture: amd64
 
 # Destination directory for RKE2 installation script
-rke2_install_script_dir: /var/tmp
+rke2_install_script_dir: "{{ rke2_install_script_dir }}"
```

Comment:
This is a loop

Reply:
You're right, nice catch. My brain must have turned off while making that commit.